cobalt/shell/app/ios-Info.plist

```diff
@@ -52,5 +52,7 @@
 	<string>Allow content_shell access to microphone</string>
 	<key>NSCameraUsageDescription</key>
 	<string>Allow content_shell access to camera</string>
+	<key>NSUserTrackingUsageDescription</key>
```

Comment:
This PR adds it here for now. But this part will be added to tvos-Info.plist, which we will add later.